client/src/app/shared/notifications-context.tsx

```diff
@@ -34,14 +34,15 @@ export const NotificationsProvider: React.FunctionComponent<
 > = ({ children }: INotificationsProvider) => {
   const [notifications, setNotifications] = React.useState<INotification[]>([]);
 
-  const pushNotification = (notification: INotification) => {
-    const hasDupe = notifications.find((n) => n.title === notification.title);
-    if (!hasDupe) {
-      setNotifications([
-        ...notifications,
-        { ...notificationDefault, ...notification },
-      ]);
-    }
+  const pushNotification = (
+    notification: INotification,
+    clearNotificationDelay?: number
+  ) => {
+    setNotifications([
+      ...notifications,
+      { ...notificationDefault, ...notification },
+    ]);
+    setTimeout(() => setNotifications([]), clearNotificationDelay || 10000);
```

Comment:
Can we replace hard coded value with a const store at higher level ?

Reply:
Any particular reason?

Reply:
For me a hard coded value deep in the application is no go.
But ultimately is if we want the user to have control over it through an option then we'll have to lift it up.

Reply:
Never mind my comment, I totally misread it :(